setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-docker build -t rgardler/acs-logging-test-base .
+docker build -t acs-logging-test-base .
```

Comment:
I suggested changing the base because when you're building one of the other images, it's easy to find it's based on the base image pulled from Docker hub (rather than a locally built version) which doesn't necessarily have the code you think it has.